### PR TITLE
fix: query bug

### PR DESCRIPTION
**Pull request by steventango**
Fixes storage not being sorted by expiry date

### CODE REVIEW

File: app/src/main/java/com/xffffff/wellfed/common/DBAdapter.java [hunk 1]
```diff
@@ -49,8 +49,7 @@ public interface OnDataChangedListener {
      * @param query the query
      */
     public DBAdapter(Query query) {
-        this.mQuery = query;
-        startListening();
+        this.setQuery(query);
```
Comment (Manpreet-Singh001) [inline app/src/main/java/com/xffffff/wellfed/common/DBAdapter.java:52]: may i know why was this required?

Comment (steventango) [reply app/src/main/java/com/xffffff/wellfed/common/DBAdapter.java:52]: It's not, but I was just trying to make it the same to figure out why it was bugging out earlier. I saw that it is cleaner, so I left it.

Comment (Manpreet-Singh001) [reply app/src/main/java/com/xffffff/wellfed/common/DBAdapter.java:52]: sounds good

**Review by Manpreet-Singh001 — APPROVED** [review]
